src/components/landscape-sidebar/geo-description-widget/geo-description-widget.js

```diff
@@ -9,17 +9,7 @@ import 'redux_modules/geo-description';
 const GeoDescriptionWidget = props => {
   const handleBackClick = () => {
     const { view } = props;
-    const { center } = view;
-    const target = {
-      position: {
-        x: center.longitude,
```

Comment:
By removing `target` object with position inside, we are messing the performance of the animation. Now we have 2 seconds delay between the camera starts moving.
This is the exact same issue as we were trying to resolve in my previous PR, check your [comment here](https://github.com/Vizzuality/half-earth-v3/pull/84#discussion_r308611055)